packages/manager/modules/overthebox/package.json

```diff
@@ -29,14 +29,15 @@
     "@ovh-ux/ng-ui-router-title": "^3.0.0",
     "@uirouter/angularjs": "^1.0.15",
     "angular": "^1.7.5",
+    "angular-translate": "^2.11.0",
     "angularjs-scroll-glue": "^2.2.0",
     "bootstrap4": "twbs/bootstrap#v4.0.0",
     "moment": "^2.24.0",
     "oclazyload": "^1.1.0",
     "ovh-angular-responsive-tabs": "^4.0.0",
-    "ovh-api-services": "^9.42.0",
+    "ovh-api-services": "^9.43.0",
```

Comment:
Could you please rebase on `develop` branch?

To make sure that you will use the latest version of `ovh-api-services` from the monorepo.

```sh
$ grep -C 1 ovh-api-services@ yarn.lock

ovh-api-services@^9.44.0:
  version "9.44.0"
```